src/breeze/components/breeze/navbar_links.cr

```diff
@@ -1,8 +1,24 @@
 class Breeze::NavbarLinks < BreezeComponent
   needs context : HTTP::Server::Context
 
+  Habitat.create do
+    setting links : Array(Proc(Breeze::NavbarLinks, Nil)) = [
+      ->(navbar : Breeze::NavbarLinks) {
+        navbar.mount_link(link_text: "Requests", link_to: Breeze::Requests::Index.path)
+      },
+      ->(navbar : Breeze::NavbarLinks) {
+        navbar.mount_link(link_text: "Queries", link_to: Breeze::Queries::Index.path)
+      },
+    ]
+  end
+
   def render
-    mount Breeze::NavbarLink(Breeze::Requests::Index), context: context, content: "Requests"
-    mount Breeze::NavbarLink(Breeze::Queries::Index), context: context, content: "Queries"
+    settings.links.each do |component|
+      component.call(self)
+    end
+  end
+
+  def mount_link(link_text : String, link_to : String)
+    mount(Breeze::NavbarLink, context: context, link_text: link_text, link_to: link_to)
```

Comment:
The idea here is that if you just want the normal navbar link that we built, you can just pass in the two values of what the link says and where it goes. But since you have access to the navbar component, you can use an escape hatch to mount your own components still.